src/handlers/uniswap/customEvents.ts

```diff
@@ -0,0 +1,132 @@
+            //             },
+            //         ]
+            //     }
+            // },
+            {
+                eventName: 'SwapEventData',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as SwapEventData;
+
+                    if (eventData.sender === eventData.recipient){
+                        return [
+                            {
+                                address: eventData.recipient,
+                                title: 'Your swap is complete',
+                                message: `The swap you initiated has been completed. Open wallet to see more details.`,
+                                options: {}
+                            }
+                        ]
+                    }
+
+                    return [
+                        {
+                            address: eventData.sender,
+                            title: 'Your swap is complete',
+                            message: `The swap you initiated has been completed`,
+                            options: {}
+                        },
+                        {
+                            address: eventData.recipient,
+                            title: 'Received Swapped Tokens',
+                            message: `You have received tokens from a swap initiated by ${eventData.sender}. Open wallet for more details.`,
+                            options: {}
+                        }
+                    ]
+                }
+            },
+            {
+                eventName: 'BurnEvent',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as BurnEventData;
+                    return [
+                        {
+                            address: eventData.owner,
+                            title: 'Removed from liquidity position',
+                            message: `Your liquidity position has been removed and a total of ${eventData.amount / parseFloat("1000000000000000")} has been returned.`,
+                            options: {}
+                        }
+                    ]
+                }
+            },
+            {
+                eventName: 'MintEvent',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as MintEventData;
+                    if (eventData.owner === eventData.sender){
+                        return [
+                            {
+                                address: eventData.owner,
+                                title: 'Liquidity Minted',
+                                message: `Your liquidity of the amount ${eventData.amount / parseFloat("1000000000000000")} has been minted`,
+                                options: {}
```

Comment:
Can we also do a call to action by providing an URL that takes the user to the page on uniswap that shows the list of liquidity pools that the user has ?

Reply:
Will try, there's limited info from the event, and it might be a little too heavy for the push notification to show but I can definitely add somethings and find out.

Reply:
@kubeCypherD added code.